cmd/config-seed/main.go

```diff
@@ -50,11 +50,8 @@ func main() {
 		os.Exit(1)
 	}
 	config.LoggingClient.Info("Service dependencies resolved...")
-	err := config.ImportProperties(dirProperties)
-	if err != nil {
-		config.LoggingClient.Error(err.Error())
-	}
-	err = config.ImportConfiguration(dirCmd, useProfile, overwriteConfig)
+
```

Comment:
Ok, will put them back.

Reply:
@tsconn23 , added this back.
Current stem used in master branch is from GlobalPrefix in config-seed toml file which is currently set to "config", not the new stem path. I am assuming this need to stay as is since the java services will be looking for "config", correct?

Reply:
Correct -- The Java services live in an older section of the Consul configuration hierarchy. Namely they live at a top-level /config directory.